src/pages/dashboard/[subdomain]/editor.tsx

```diff
@@ -819,10 +835,8 @@ const EditorExtraProperties: FC<{
           label={t("Tags") || ""}
           id="tags"
           isBlock
-          onChange={(e: ChangeEvent<HTMLInputElement>) =>
-            updateValue("tags", e.target.value)
```

Comment:
Removing this code will result in the visibility not being updated to "Modified" after tags are updated, causing the publish button to remain disabled.